src/resources/views/crud/fields/relationship/fetch_or_create.blade.php

```diff
@@ -269,7 +269,13 @@ function setupInlineCreateButtons(element) {
 
         //prepare main form fields to be submited in case there are some.
         if(typeof $includeMainFormFields === "boolean" && $includeMainFormFields === true) {
+
+            // we trigger this event so that fields in need to parse their value before sent in some ajax request,
+            // case of repeatable, can catch it and be sent with main form.
+            $form.trigger('backpack_field.parse_value', element);
```

Comment:
@pxpm after a huge debug, I found out `fetch_or_create` was not working properly because this code is inside `$inlineCreateButtonElement.on('click'` function, and it should be inside `element.select2({data:` function.

**I added it below.**
I'm not sure if this code is necessary here for anything related with inline create, can you make sure?

Reply:
@promatik indeed, you are right, it should be inside the `element.select2` too. 

- One is the regular field when doing the search for something to select (`include_all_form_fields`) in the main field.
-  the other is when you make the request for inline create modal,
 ```php
'inline_create' => [
    'include_main_form_fields' => true
]
``` 
Nice catch!